dbgpt/storage/knowledge_graph/community_summary.py

```diff
@@ -363,6 +391,7 @@ async def asimilar_search_with_scores(
                     limit=self._knowledge_graph_chunk_search_top_size,
                     search_scope="document_graph",
                 )
+
```

Comment:
We need an additional vector search of chunk.content and trace back all the upstream paths of chunks to obtain a document graph based on chunks and merge it with the document graph based on triple subgraph search.